float32 avg_dist
float32 rel_vel
float32 control_signal

